docs/modules/kafka_ingester.md

```diff
@@ -0,0 +1,59 @@
+# Kafka ingester
+
+|                |                         |
+|----------------|-------------------------|
+| `moduleName`   | `kafkaIngester`         |
+| Module type    | `producer`              |
+| Output event   | `raw`                   |
+
+Kafka ingester generates events from Kafka messages.
+
+`moduleConfig`
+```yaml
+# Allow verbose logging of events within Kafka library. Global logger with its configured log level is used.
+logKafkaEvents: false
+# Allow logging of errors within Kafka library. Global logger with its configured log level is used.
+logKafkaErrors: true
+# List of Kafka brokers
+brokers:
+  - <string> # e.g. kafka-1.example.com:9092
+topic: ""
+groupId: ""
+# commitInterval indicates the interval at which offsets are committed to the broker.
+# If 0 (default), commits will be handled synchronously.
+commitInterval: <duration> # e.g. 0, 5s, 10m
+# retentionTime optionally sets the length of time the consumer group will be saved by the broker.
+# Default: 24h
+retentionTime: <duration>
+# lastOffsetAsDefault determines from whence the consumer group should begin consuming when it finds a partition without a committed offset.
+# If true, partition will be read from its LastOffset, otherwise from the FirstOffset.
+lastOffsetAsDefault: false
```

Comment:
I'm aware of that this is not most understandable option name, but was the best I could come with.

Suggestions welcomed. :)

Reply:
maybe consider: `StartOffset` (or better `KafkaStartOffset`) with enum of allowed values: `FirstOffset`, `LastOffset`.
But I am not sure that his is much better.

Reply:
Hm.. the enum might be a way to go.  Something as `offsetToUse: firts|last` even though the validation will be a bit more complicated :/

Reply:
Changed within fca8166

Now it is named `fallbackStartOffset` (I was considering also `defaultStartOffset`).

TBH `StartOffset,offsetToUse` seemed quite misledading to me as it is not clear from the name that it does apply only at certain condition (when there is a partition with no commited offset for a given groupid)